src/config/sqs.ts

```diff
@@ -1,12 +1,23 @@
 import * as AWS from 'aws-sdk';
+import https from 'https';
 import { Configuration } from '../common';
 
 const sqs = (config: Configuration) => {
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
@anupvarghese rather than have the env check, can we add it to configuration

Reply:
ok 👍